lms/djangoapps/course_api/serializers.py

```diff
@@ -74,6 +74,7 @@ class CourseSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     mobile_available = serializers.BooleanField()
     hidden = serializers.SerializerMethodField()
     invitation_only = serializers.BooleanField()
+    license = serializers.CharField()
```

Comment:
Please update the endpoint's docstring to include/explain the license field, and include in the `Example response`:
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_api/views.py#L17

Reply:
I'd rather we focus on using an actual documentation library like Swagger or DRF's core API than continue to use docstrings (which I clearly glossed over).

Reply:
I've started a branch at https://github.com/edx/edx-platform/tree/clintonb/api-docs, but the documentation doesn't work as expected, perhaps due to the forked version of DRF, our somewhat unorthodox method of using `urls.py`, or something else. API documentation for edx-platform is a low priority for me, but others are welcome to pick up the branch and finish the work.